contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-react-native.mdx

```diff
@@ -83,7 +83,7 @@ You can configure the `featureFlagsRequestTimeoutMs` parameter when initializing
 ```react-native
 export const posthog = new PostHog('<ph_project_api_key>', {
   // usually 'https://us.i.posthog.com' or 'https://eu.i.posthog.com'  
-  host: '<ph_client_api_host>', // TIP: host is optional if you use https://us.i.posthog.com
```

Comment:
`TIP` is a feature, why remove it? we have it in a lot of places btw

Reply:
I kept it in the places where the docs focus on installation, but in this scenario its a distraction. We're talking about how to use posthog for feature flags, so we dont need to draw attention to this TIP (the blue dot is quite distracting IMO)

Reply:
In this case, I'd even remove the `host` altogether then, it's also an optional parameter that should have been set in the installation as well.

Reply:
yeah usually its a copy-pasta line haha